pkg/services/authn/authnimpl/service.go

```diff
@@ -320,6 +320,15 @@ func (s *Service) RegisterClient(c authn.Client) {
 	}
 }
 
+func (s *Service) IsClientEnabled(name string) bool {
+	client, ok := s.clients[name]
```

Comment:
Could you please add a comment to describe the expected client name template? Something like that the name should start with `auth.client.*` + maybe adding an example to the comment that for OAuth providers the name should be `auth.client.{providerName}` + the `authn.ClientWithPrefix(name)` helper function could be used for generating the name.

Reply:
I added some comments and provided some examples for SAML and for OAuth clients as requested. LMK what you think!